server/join_test.go

```diff
@@ -53,24 +53,33 @@ func newTestMultiJoinConfig(count int) []*Config {
 
 func waitMembers(svr *Server, c int) error {
 	// maxRetryTime * waitInterval = 5s
-	maxRetryCount := 500
-	waitInterval := 100 * time.Millisecond
+	maxRetryCount := 10
+	waitInterval := 500 * time.Millisecond
+	ctx, cancel := context.WithTimeout(context.Background(), defaultDialTimeout)
+	defer cancel()
+
+	client := svr.GetClient()
+	for ; maxRetryCount != 0; maxRetryCount-- {
```

Comment:
@siddontang  Seems not, c means the waited member count, but maxRetryCount is only for retry count of request, not the same here.